### DISCUSSION TITLE
Receiving the folowing error while trying to to add Darktrace Integration

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#10194] [2.4] [answered]
Error
Error: cannot use 'in' operator to search for "attributes" in "404 page not fou..."
o@https://replaced/kibana/59020/bundles/plugin/fleet/kibana/fleet.plugin.js:2:12053
m/<@https://replaced/kibana/59020/bundles/plugin/fleet/kibana/fleet.plugin.js:2:81400
qu@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:106314
__kbnSharedDeps_npm__</t.unstable_runWithPriority@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:406:3768
$i@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:44909
Uu@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:105759
xu@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:96253
Zi/<@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:45132
__kbnSharedDeps_npm__</t.unstable_runWithPriority@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:406:3768
$i@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:44909
Zi@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:45079
Ki@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:45012
Su@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:93679
Da@https://replaced/kibana/59020/bundles/kbn-ui-shared-deps-npm/kbn-ui-shared-deps-npm.dll.js:398:64508
Ee/w<@https://replaced/kibana/59020/bundles/plugin/esUiShared/kibana/esUiShared.plugin.js:1:54742

**Comment by dougburks** [comment]
What version of Security Onion are you using?

What are the exact set of steps that led to this?

**Comment by Phillip-Samson** [comment, marked as answer]
Apologies, Doug
2.4 is what I am using but this issue has seemed to remedy itself. Please disregard.
